fairchive-webapp/src/main/java/edu/harvard/iq/dataverse/settings/SettingsServiceBean.java

```diff
@@ -798,7 +798,16 @@ public enum Key {
          * pre-filled on the collection creation form. If set to {@code false}
          * or any value other non-truthy value, the fields will be blank.
          */
-        CollectionFormPrefillEnabled;
+        CollectionFormPrefillEnabled,
+
+        /**
+         * If set to {@code true} (the default) or any other truthy value,
+         * the button for adding dataset will be rendered on dataverse page
+         * (alongside with add dataverse button). If set to {@code false}
+         * or any value other non-truthy value, the only way to add a new
```

Comment:
```suggestion
         * or any other non-truthy value, the only way to add a new
```

I've made a similar mistake above in the description of `CollectionFormPrefillEnabled`. Could you pleas fix that as well?